apps/docs/content/tokens/semantic/elevation.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Elevation
+description: Getting started with Workleap Design Tokens
+---
+import tokens from '../../../datas/tokens.json';
+
+# Elevation
+
+## Guideline
+
+### Working with elevation
+
+Elevation allows to bring a sense of materiality and depth to the interface. Use elevation to infer information hierarchy.
+
+### Don’t skip elevations
+
+When applying elevation to visual elements, don’t skip elevations so that users understand the layering of elements on the page. Starting with no elevation at the background level, build interfaces by “piling up” containers and other elements on top of each other.
+
+### Use with purpose
+
+Each elevation has a purpose and allow users to see what is closer to them – what requires their attention now. Always respect how elements are stacked on top of the other so that users understand the materiality of the app.
```

Comment:
```suggestion
Each elevation has a purpose and allow users to see what is closer to them – what requires their attention now. Always respect how elements are stacked on top of each other so that users understand the materiality of the app.
```